model/fieldtypes/compat/Float.php

```diff
@@ -0,0 +1,6 @@
+<?php
+
```

Comment:
Should we put a conditional on PHP_MAJOR_VERSION in here too? I'd move it to the start of this code from the DBFLoat.php file.


```
<?php

if (PHP_MAJOR_VERSION < 7) {
	return;
}

class Float extends DBFloat {
}
```